src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+  }
+
+  static addAnimal(animal) {
+    Animal.alive.push(animal);
+  }
+
+  static removeAnimal() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
```

Comment:
The method name `removeAnimal` suggests a single animal is being removed, but the implementation removes all dead animals. Consider renaming the method to reflect its functionality, such as `removeDeadAnimals`.